wdl-engine/src/inputs.rs

```diff
@@ -616,18 +646,50 @@ impl Inputs {
         let reader = BufReader::new(file);
         let object = mem::take(
             serde_json::from_reader::<_, JsonValue>(reader)
-                .with_context(|| {
-                    format!("failed to parse input file `{path}`", path = path.display())
-                })?
+                .context("failed to parse input file")?
                 .as_object_mut()
-                .with_context(|| {
-                    format!(
-                        "expected input file `{path}` to contain a JSON object",
-                        path = path.display()
-                    )
-                })?,
+                .context("expected input file to contain a JSON object")?,
```

Comment:
Similar to the above comment, I think we should keep the `with_context` call here.